packages/screens/RiotGame/component/FightCountdownSection.tsx

```diff
@@ -1,130 +1,159 @@
 import moment from "moment";
-import React from "react";
-import { Image, StyleSheet, TouchableOpacity, View } from "react-native";
+import React, { useMemo } from "react";
+import {
+  Image,
+  ImageBackground,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from "react-native";
 
+import brokenBoxPNG from "../../../../assets/game/broken-box.png";
 import clockSVG from "../../../../assets/game/clock.svg";
 import countDownPNG from "../../../../assets/game/countdown.png";
 import unstakeSVG from "../../../../assets/icons/unstake.svg";
 import { BrandText } from "../../../components/BrandText";
-import FlexRow from "../../../components/FlexRow";
 import { SVG } from "../../../components/SVG";
-import { TertiaryBox } from "../../../components/boxes/TertiaryBox";
-import { SpacerRow } from "../../../components/spacer";
-import { GetSquadResponse } from "../../../contracts-clients/teritori-squad-staking/TeritoriSquadStaking.types";
+import { SpacerColumn } from "../../../components/spacer";
+import { Squad } from "../../../contracts-clients/teritori-squad-staking/TeritoriSquadStaking.types";
 import { StakingState } from "../../../utils/game";
 import {
-  neutral33,
   neutral77,
   redDefault,
   yellowDefault,
 } from "../../../utils/style/colors";
 import {
-  fontMedium48,
   fontSemibold20,
   fontSemibold14,
+  fontMedium40,
 } from "../../../utils/style/fonts";
 import { layout } from "../../../utils/style/layout";
 
 type FightCountdownSectionProps = {
-  isUnstaking: boolean;
-  remainingTime: number;
-  currentSquad: GetSquadResponse | undefined;
-  stakingState: StakingState;
   unstake(): void;
+  isUnstaking: boolean;
+  squad: Squad;
+  now: number;
+  cooldown: number;
 };
 
 export const FightCountdownSection: React.FC<FightCountdownSectionProps> = ({
-  isUnstaking,
-  remainingTime,
-  currentSquad,
-  stakingState,
   unstake,
+  isUnstaking,
+  squad,
+  now,
+  cooldown,
 }) => {
+  const { remainingTime, stakingState } = useMemo(() => {
+    let _remainingTime = 0;
+    let _stakingState = StakingState.UNKNOWN;
```

Comment:
why _ prefix?

Reply:
It was used to differentiate with remainingTime, stakingState from parent scope just to make the code more readable but I can remove _, there is no problem.